client/actions/auth.js

```diff
@@ -60,10 +59,11 @@ export const loginUser = (user) => {
           errorMessage = 'Internet connection error';
         } else if (error.response.data.name === 'TimeoutError') {
           errorMessage = 'Request timed out';
```

Comment:
I noticed you are not returning proper error messages for errors. Try to return proper error message.